mkdocs/docs/HPC/torque_frontend_via_jobcli.md

```diff
@@ -0,0 +1,131 @@
+# Torque frontend via jobcli
+
+## What is Torque
+
+[Torque](https://en.wikipedia.org/wiki/TORQUE) is a resource manager for submitting and managing jobs on an HPC cluster. It is and implementation of [PBS (Portable Batch System)](https://en.wikipedia.org/wiki/Portable_Batch_System). Torque isn't widely used anymore, so the {{hpcinfra}} no longer uses Torque in the backend, however the commands were kept to avoid researchers having to learn other commands.
+
+## Slurm backend
+
+Just like Torque, [Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager) is a resource manager for submitting and managing jobs on an HPC cluster. Currently, Slurm is the most popular workload manager on HPC systems worldwide, however it makes use of different commands that are less intuitive and user friendly than Torque/PBS.
+
+## jobcli
+
+Jobcli was developed to make it possible for the {{hpcinfra}} to use a Torque frontend and a Slurm backend. In addition to that, it adds some additional options for Torque commands.
```

Comment:
```suggestion
Jobcli is a Python library that was developed by {{hpcteam}} to make it possible for the {{hpcinfra}} to use a Torque frontend and a Slurm backend. In addition to that, it adds some additional options for Torque commands.
```

Reply:
This needs a bit more meat: it should also mention that via `jobcli` the commands of Torque user interface (`qsub`, etc. <list them all>) are made available to end users. These correspond to Python script that "translate" the Torque commands to the equivalent Slurm commands, and in the case of `qsub` also make some changes to the provided job script to make it compatible with Slurm (like translating `#PBS` lines to the equivalent `#SBATCH` lines).